### PR TITLE
feat: cleanup CRA in DMS Demo

### LINKED ISSUE #254: [ENHANCEMENT] Clean up after cra in dms
# 🙋 Feature Request

We should clean up after Create React App in dms-demo.

## ⚙ Tasks

- [x] Remove config, scripts folders (we don't use them)
- [x] Review dependecies (cra puts everything to dependecies. Remove those that we don't use and move those that are dev dependecies to devDependecies)

### PR DESCRIPTION
**Pull request by RanadeepPolavarapu**
Resolves #254

Signed-off-by: Ranadeep Polavarapu <ranadeep@icloud.com>

### CODE REVIEW

File: examples/sn-dms-demo/package.json [hunk 1]
```diff
@@ -67,49 +60,58 @@
...
     "cypress": "^3.4.1",
+    "dotenv": "^8.1.0",
+    "file-loader": "4.2.0",
+    "html-webpack-plugin": "^3.2.0",
     "path-to-regexp": "3.0.0",
+    "postcss-flexbugs-fixes": "4.1.0",
+    "postcss-loader": "^3.0.0",
     "raw-loader": "^3.1.0",
     "redux-mock-store": "^1.5.3",
+    "reflect-metadata": "^0.1.12",
```
Comment (B3zo0) [inline examples/sn-dms-demo/package.json]:
```suggestion
```

File: examples/sn-dms-demo/package.json [hunk 2]
```diff
@@ -67,49 +60,58 @@
...
     "@types/redux-mock-store": "^1.0.1",
     "@types/uuid": "^3.4.5",
+    "autoprefixer": "^9.6.1",
     "awesome-typescript-loader": "^5.2.1",
     "chance": "^1.0.18",
     "copy-webpack-plugin": "^5.0.4",
     "cross-env": "^5.2.0",
+    "css-loader": "^3.2.0",
     "cypress": "^3.4.1",
+    "dotenv": "^8.1.0",
```
Comment (B3zo0) [inline examples/sn-dms-demo/package.json]:
```suggestion
```

File: examples/sn-dms-demo/package.json [hunk 3]
```diff
@@ -67,49 +60,58 @@
...
     "redux-di-middleware": "^4.0.1",
-    "reflect-metadata": "^0.1.12",
+    "redux": "^4.0.4",
     "semaphore-async-await": "^1.5.1",
     "ts-keycode-enum": "^1.0.6",
     "typeface-roboto": "^0.0.75",
     "uuid": "^3.3.3"
   },
   "devDependencies": {
+
```
Comment (B3zo0) [inline examples/sn-dms-demo/package.json]:
```suggestion
```

File: examples/sn-dms-demo/package.json [hunk 4]
```diff
@@ -43,20 +44,12 @@
     "@sensenet/redux": "^5.1.13",
     "@sensenet/repository-events": "^1.4.5",
     "@sensenet/search-react": "^1.2.9",
-    "autoprefixer": "^9.6.1",
-    "css-loader": "^3.2.0",
-    "dotenv": "^8.1.0",
-    "file-loader": "4.2.0",
     "fs-extra": "^8.1.0",
```
Comment (B3zo0) [inline examples/sn-dms-demo/package.json]:
```suggestion
```

File: examples/sn-dms-demo/package.json [hunk 5]
```diff
@@ -28,6 +28,7 @@
     "fix:prettier": "prettier \"{,!(dist|temp|bundle)/**/}*.{ts,tsx}\" --write"
   },
   "dependencies": {
+
```
Comment (B3zo0) [inline examples/sn-dms-demo/package.json]:
```suggestion
```

**Comment by RanadeepPolavarapu** [comment]
@B3zo0 Done. Resolved conflicts and resolved your suggestions.

**Pull request merged**